plugins/processors/filter/sample.conf

```diff
@@ -0,0 +1,27 @@
+# Filter metrics by the given criteria
+[[processors.filter]]
+    ## Default action if no rule applies
+    # default = "pass"
+
+    ## Rules to apply on the incoming metrics (multiple rules are possible)
+    ## The rules are evaluated in order and the first matching rule is applied.
+    ## In case no rule matches the "default" is applied.
+    ## All filter criteria in a rule must apply for the rule to match the metric
+    ## i.e. the criteria are combined by a logical AND. If a criterion is
+    ## omitted it is NOT applied at all and ignored.
+    [[processors.filter.rule]]
+        ## List of metric names to match including glob expressions
+        # name = []
+
+        ## List of tag key/value pairs to match (no glob expressions)
+        ## ALL given tags must exist for the metric to match the rule.
+        # tags = {}
```

Comment:
Too bad there is no support for glob in the tags. Any reason? Or should that be tackled with `tagpass` and `tagdrop`?

Reply:
Will try to work on this...